docs/tutorials/faq.md

```diff
@@ -202,12 +202,12 @@ supported.  This supports both DASH and HLS manifests.
 **Q:** The Nightly Demo isn't loading for me!
 
 **A:** Are you looking at the uncompiled build with an AdBlocker enabled?
-We're rolling out ad support which is triggering some ad blockers to block
+We're rolling out ad support which is triggering some adblockers to block
```

Comment:
Hm. Is "adblocker" considered a word now? Like, the [wikipedia article on the subject](https://en.wikipedia.org/wiki/Ad_blocking) mostly keeps it at two words.

Reply:
The article also has `adblocking` mentioned, so I think both are the right conventions. If it's not an issue, we can keep adblockers instead of ad blocker or vice versa.